### PR TITLE
Enhancement (ci): Add `commit_sha` label

### PR DESCRIPTION
**Pull request by leojonathanoh**
This is useful for traceability.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -79,6 +79,10 @@ else
     export STEAM_USERNAME=${STEAM_USERNAME:-}
     export STEAM_PASSWORD=${STEAM_PASSWORD:-}
 fi
+COMMIT_SHA=${COMMIT_SHA:-${CIRCLE_SHA1:-}} # CircleCI
+COMMIT_SHA=${COMMIT_SHA:-${GITHUB_SHA:-}} # GitHub CI
+COMMIT_SHA=${COMMIT_SHA:-${CI_COMMIT_SHA:-}} # GitLab CI
+COMMIT_SHA=${COMMIT_SHA:-$( git rev-parse HEAD )} # Get from git
```
Comment (joeltimothyoh) [inline build.sh:85]: Better to just use `git rev-parse HEAD` alone so it's independent from proprietary CI variables

**Review by joeltimothyoh — CHANGES_REQUESTED** [review]

File: build.sh (hunk 1 above)
Comment (leojonathanoh) [reply build.sh:85]: not every CI has `git` installed.

Comment (joeltimothyoh) [reply build.sh:85]: either way this only adds the dependency which might be better done without

Comment (joeltimothyoh) [reply build.sh]: with `docker:20.10.18-git` image on gitlab retrieving from `git` itself would be fine

**Review by joeltimothyoh — APPROVED** [review]

**Pull request merged**